docs/topics/db/fixtures.txt

```diff
@@ -126,11 +126,31 @@ You could also write a decorator to encapsulate this logic::
 Just be aware that this logic will disable the signals whenever fixtures are
 deserialized, not just during ``loaddata``.
 
-Note that the order in which fixture files are processed is undefined. However,
-all fixture data is installed as a single transaction, so data in
-one fixture can reference data in another fixture. If the database backend
-supports row-level constraints, these constraints will be checked at the
-end of the transaction.
+Fixtures can be loaded in a specified order. For example:
+
+.. code-block:: shell
+
+    manage.py loaddata foo bar baz
+
+or (in a test case)
+
+.. code-block:: shell
+
+    Fixtures = ['foo', 'bar', 'baz']
+
+In this example, ``foo`` refers to any fixture named ``foo`` in any application.
+This would load ``foo`` followed by ``bar`` and then ``baz``.
```

Comment:
```suggestion
This would first load all the ``foo``-named fixtures from all applications (in
the order in which applications are defined), followed by all the ``bar`` and
then all the ``baz`` fixtures.
```